css/style.css

```diff
@@ -0,0 +1,73 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  font-family: "oppins", sans-serif;
+  background-image: url("../img/background.png");
+  background-repeat: no-repeat;
+}
+
+.container {
+  margin: 5%;
+  padding-top: 3rem;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 1em 1.8em;
+}
+
+.logo {
+  font-weight: 700;
+  font-size: 1.2rem;
+  line-height: 1.4rem;
+  text-align: center;
+  color: #6070ff;
+}
+
+.bars {
+  color: #6070ff;
+}
+
+.hero-main-text {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 2rem;
+  line-height: 2rem;
+  color: #172b4d;
+}
+
+.hero-minor-text {
+  color: #344563;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 1rem;
+  line-height: 1.8rem;
+}
```

Comment:
- [ ] Kindly give space between the main text and the smaller text. I would recommend you give a margin of `12px` between the elements. 👍 💻 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/221051071-1a579de2-1fd0-4274-8d15-613acf3fa51b.png)

### Figma Design

![image](https://user-images.githubusercontent.com/77106962/221051141-ea9d48dd-853a-41c8-8beb-8925cb2ebefb.png)

Reply:
done :heavy_check_mark: